portability-transfer/src/main/java/org/datatransferproject/transfer/WorkerModule.java

```diff
@@ -197,4 +206,22 @@ Monitor getMonitor() {
   ExtensionContext getContext() {
     return context;
   }
+
+  private TransferServiceConfig getTransferServiceConfig(TransferExtension ext) {
+    String configFileName = "config/" + ext.getServiceId().toLowerCase() + ".yaml";
+    InputStream inputStream = this.getClass().getClassLoader().getResourceAsStream(configFileName);
+    getMonitor().info(
+        () -> "Service %s has a config file: %s",
+        ext.getServiceId(),
+        (inputStream != null));
+    if (inputStream == null) {
+      return TransferServiceConfig.getDefaultInstance();
```

Comment:
Let's log: using default transfer config for serviceId ...

Reply:
This is logged about the if statement.